homeassistant/components/climate/nest.py

```diff
@@ -97,6 +97,20 @@ def __init__(self, structure, device, temp_unit):
         self._min_temperature = None
         self._max_temperature = None
 
+    @property
+    def should_poll(self):
+        """Do not need poll thanks using Nest streaming API."""
+        return False
+
+    async def async_update_state(self):
```

Comment:
Can you define this function inside `async_added_to_hass`, I wouldn't want to have it accidentally clash with a future function on entity with a similiar name.